Rubberduck.Inspections/Inspector.cs

```diff
@@ -91,7 +91,7 @@ private void UpdateInspectionSeverity(Configuration config)
                     LogManager.GetCurrentClassLogger().Error(e);
                 }
 
-                var issuesByType = allIssues.GroupBy(issue => issue.GetType())
+                var issuesByType = allIssues.GroupBy(issue => issue.Inspection.Name)
```

Comment:
Did we have inspections that returned different result types? If so, results from these inspections are no longer correctly aggregated. Although, less badly than before this fix...